src/dev/impl/DevToys/Views/Tools/Converters/CronParser/CronParserToolPage.xaml

```diff
@@ -102,7 +103,15 @@
              Tag="UserCronExpression"/>
 
         <controls:CustomTextBox
-             Grid.Row="2"
+            Grid.Row="2"
+            IsReadOnly="True"
+            AcceptsReturn="True"
```

Comment:
I think this should be false :)

Reply:
I left this as `True` because it affects the UI. When it is `True`, the copy button is placed above the text box and aligns nicely with the rest of the page.

| ![image](https://github.com/veler/DevToys/assets/7417301/71f64368-abaf-480d-89e9-fa1d356ebc30) |
| - |

When it is `False`, the copy button is placed inline and doesn't look as nice.

| ![image](https://github.com/veler/DevToys/assets/7417301/bbc74dd6-c3da-4204-9a0e-95fbf32195b3) |
| - |

Since it's a readonly textbox, I figured `AcceptsReturn` won't have any negative affect.

What do you think?

Reply:
That's fair! Let's keep it True then :)